### PR TITLE
Support linux for kind clusters s/x86_64/amd64

### PR DESCRIPTION
**Pull request by srust**

### CODE REVIEW

**Review by luthermonson — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -203,7 +203,11 @@ $(TILT): $(LOCALBIN)
 kind: $(KIND) ## Download kind locally if necessary. If wrong version is installed, it will be overwritten.
 $(KIND): $(LOCALBIN)
 	test -s $(KIND) && $(KIND) version | grep -q $(KIND_VERSION) || \
-	(cd $(LOCALBIN); curl -Lso ./kind https://kind.sigs.k8s.io/dl/v$(KIND_VERSION)/kind-$(OS)-$(ARCH) && chmod +x kind)
+	(KIND_ARCH=$(ARCH); \
```
Comment (luthermonson) [inline Makefile:206]: maybe better to change ARCH to GOARCH?

Comment (srust) [reply Makefile:206]: you mean do something like:

```
KIND_ARCH=$(shell go env GOARCH)
```
?

Comment (luthermonson) [reply Makefile:206]: ya, that

**Comment by eljohnson92** [comment]
This is now handled as part of #65

**Pull request closed without merging**